eng/fv/appendix-fv.rst

```diff
@@ -0,0 +1,1318 @@
+.. SPDX-License-Identifier: CC-BY-SA-4.0
+
+.. Copyright (C) 2022 Trinity College Dublin
+
+Appendix: RTEMS Formal Model Guide
+**********************************
+
+Here we describe the various formal models of RTEMS that are currently in
+existence. The following were delivered as part of the ESA-sponsored activity,
+and are included in the initial set when the directory ``formal`` was added
+to ``rtems-central``:
+
+Chains API (ESA-sponsored Activity)
```

Comment:
Not appropriate to specifically call  out sponsors in sections of our manuals.

Reply:
Fair enough.